src/main/jbake/content/blog/2017/03/14/hawkular-alerting-tutorial.adoc

```diff
@@ -0,0 +1,34 @@
+= Hawkular Alerting: Tutorial Now Available!
+Jay Shaughnessy
+2017-03-14
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, alerting
+
+Hawkular Alerting is a pluggable, API-driven component for adding alerting to your application or tool.  It's one of the many http://hawkular.org[Hawkular] offerings for monitoring and management.  To help quick-start new users, and as a reference for current users, we've released the first five lessons of a new tutorial, with more to follow.  It's a great way to hit the ground running.
+
+====== For Git Users:
+
+----
+git clone https://github.com/hawkular/hawkular-alerts.git
+cd hawkular-alerts/examples/tutorial
+----
+
+====== For Others:
+
+. Click this link to visit http://github.com/hawkular/hawkular-alerts[Hawkular Alerting on GitHub].
+. Click _Clone or download_
+. Click _Download ZIP_
+
+----
+unzip hawkular-alerts-master.zip
+cd hawkular-alerts/examples/tutorial
+----
+
+Start with the README.adoc. If you don't have an asciidoc extension in your browser you can read the rendered README https://github.com/hawkular/hawkular-alerts/blob/master/examples/tutorial/README.adoc[directly from GitHub].  From here just click links in the document to proceed.
```

Comment:
Add a section heading before this to guide the reader that the "common" instructions continue here